### PR TITLE
Fix tests for voxel filter.

### LINKED ISSUE #2940: filters.volxeldownsize tests don't work
Some tests don't really check anything and pass by not processing any data.

### PR DESCRIPTION
**Pull request by abellgithub**
Close #2940

### CODE REVIEW

File: doc/stages/filters.voxeldownsize.rst [hunk 1]
```diff
@@ -39,9 +40,11 @@ cell
...
+  **voxelcenter**: Coordinates of the first point found in each voxel will
+  be modified to be the center of the voxel.
+  **firstinvoxel**: Only the first point found in each voxel is retained.
 
 .. warning::
-    If you choose **voxelcenter** mode, you are loosing your dimensional data/point coordinates.
-  
\ No newline at end of file
+    If you choose **voxelcenter** mode, you are overwriting the X, Y and Z
+    values of retained points.
```
Comment (chambbj) [inline doc/stages/filters.voxeldownsize.rst]: Originally he says "losing your dimensional data", which I suppose isn't true, the dimension still exists, but it is invalidated in the sense that it is simply inheriting the dimension data of the first point, which may or may not be a good thing. Maybe we should still try to capture this fact in some manner.

Comment (abellgithub) [reply doc/stages/filters.voxeldownsize.rst]: Updated.  Let me know if you have better wording.

File: doc/stages/filters.voxeldownsize.rst [hunk 2]
```diff
@@ -39,9 +40,11 @@ cell
 
 mode
   Mode for voxel based filtering. [Default: voxelcenter]
-  **voxelcenter**: Retain center of a populated voxel. Coordinates of a first point detected will be modified and set to the center of a populated voxel.
-  **firstinvoxel**: Retain first point detected in each voxel.
+  **voxelcenter**: Coordinates of the first point found in each voxel will
+  be modified to be the center of the voxel.
+  **firstinvoxel**: Only the first point found in each voxel is retained.
```
Comment (chambbj) [inline doc/stages/filters.voxeldownsize.rst]: Docs show these as `voxelcenter` or `firstinvoxel` with the former being the default. The updated code indicates the choices would be `center` or `first` with `center` still being the default. A call to PDAL options however displays the following message, which is misleading.

```
$ pdal --options=filters.voxeldownsize
filters.voxeldownsize -- http://pdal.io/stages/filters.voxeldownsize.html
-----------------------------------------------------------------------------------------------------
  user_data
      User JSON

  log
      Debug output filename

  option_file
      File from which to read additional options

  cell [0.001]
      Cell size

  mode [centerfirst]
      Method for downsizing : center / first
```

Comment (abellgithub) [reply doc/stages/filters.voxeldownsize.rst]: Oops

**Pull request merged**